### PR TITLE
feat: docsify to docusaurus

### LINKED ISSUE #217: Feature: transfer course to docusaurus
### Suggested solution

_No response_

### PR DESCRIPTION
**Pull request by BekahHW**
## Description

Moved all the content over to docusaurus to prep for making it a subdirectory.

### Needs done

- [x] Change the Tips to [Admonitions](https://docusaurus.io/docs/next/markdown-features/admonitions)
- [x] Double check that all files were moved over
- [x] Remove the "edit this page footer"

## Related Tickets & Documents

closes #217 

## Mobile & Desktop Screenshots/Recordings

image

## Steps to QA

https://deploy-preview-222--sauced-intro.netlify.app/

## Tier (staff will fill in)

- [ ] Tier 1
- [x] Tier 2
- [ ] Tier 3
- [ ] Tier 4

### CODE REVIEW

**Review by adiati98 — COMMENTED** [review]
@BekahHW, this is great! Everything works as it should! 🎉 

I left some reviews here.

There's also one thing. The blog link on the navbar leads to the OpenSauced blog and the other on footer leads to DEV. Is this on purpose?

If you need me to, I can get through the checklist and fix them. Just let me know. Thanks!

File: sidebars.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+          'becoming-a-maintainer/how-to-setup-your-project',
+          'becoming-a-maintainer/building-community',
+          'becoming-a-maintainer/communication-and-collaboration',
+          'becoming-a-maintainer/issues-and-pull-requests',
+          'becoming-a-maintainer/maintaining-code-quality',
+          'becoming-a-maintainer/maintainer-powerups',
+          'becoming-a-maintainer/maintainers-guestbook',
+          'becoming-a-maintainer/metrics-and-analytics',
+          'becoming-a-maintainer/your-team',
+          'becoming-a-maintainer/additional-resources',
```
[the remark this answers is not in the document]
Comment (BekahHW) [reply sidebars.js:42]: We should definitely have them in the same order. I wrote a script to move everything over to the right format, so I must not have told it to maintain the current order.

[the remark this answers is not in the document]
Comment (BekahHW) [reply sidebars.js]: I fixed it!

File: docs/intro-to-oss/README 2.md
[the remark this answers is not in the document]
Comment (BekahHW) [reply docs/intro-to-oss/README 2.md:1]: We actually didn't need this file. The info is on pages>index.js

**Comment by BekahHW** [comment]
@adiati98 I think everything is there an where it needs to be. Two things that could be done:

1. We need to decide what to do with the translations. It looks like both of those discussions have support. So we might just need to provide them with instructions. I did remove the section on translations in here, so we should probably add info in the main readme.
2. The key words are just the titles with Open Source and Open Source community. Ideally, we'd be more specific with each page. 

Ideally, we get this merged in sooner rather than later. So we might need

**Comment by adiati98** [comment]
> @adiati98 I think everything is there an where it needs to be. Two things that could be done:
> 
> 1. We need to decide what to do with the translations. It looks like both of those discussions have support. So we might just need to provide them with instructions. I did remove the section on translations in here, so we should probably add info in the main readme.
> 2. The key words are just the titles with Open Source and Open Source community. Ideally, we'd be more specific with each page.
> 
> Ideally, we get this merged in sooner rather than later. So we might need

Hey @BekahHW,

I'm no longer on my computer. So I'll check these tomorrow. 🙂 

Quick responding to you here:

1. Yes, I believe both translations have support. I don't think we need to add any info in the main README. We do, however, need to update the translation guidelines (issue #212) to completely hold all information about community translations, including the instructions.
2. We can create another issue for the keywords and work on it later if we want this to be merged fast. That way, we can think through of suitable keywords to put in each page.

Also, your comment is cut off. Is there any continuation there? 😅

**Comment by adiati98** [comment]
@BekahHW,

I hope you don't mind that I made some fixes here:

- Replaced docsify to Docusaurus admonitions
- Fixed spacings for admonitions
- Added maintainer glossary to the sidebar
- Moved `h1` to the `title` on front matter to fix [multiple top-level headings in the same document](https://github.com/DavidAnson/markdownlint/blob/v0.34.0/doc/md025.md) warning.
- Fixed links in main README.

**Suggestion:**

We have an intro chapter at intro to OSS course. I propose to:

- Delete the `intro.md` file because it's almost duplication with the content in the README. And also because we don't have this chapter at Becoming A Maintainer course. So we can keep them both consistent.
- If this proposal is a go, we need to remove the "Intro" subsection of "Course Overview" section on the intro to OSS README.

FYI, I'm working on adding keywords atm. Hopefully, we still can merge this in by today. 🙂

**Review by adiati98 — APPROVED** [review]
LGTM! 🚀 

woohoo gif

**Review by nickytonline — APPROVED** [review]
Loaded the site and everything seems to be working.

**Pull request merged**